src/Visualization/Python/GenerateXdmf.py

```diff
@@ -8,9 +8,22 @@
 import h5py
 import logging
 import numpy as np
+import rich
 import sys
 
 
+def available_subfiles(h5file):
```

Comment:
This is a duplicate (except for the `".vol"` part) of the `available_subfiles` function in `PlotDatFile.py`. Maybe we should combine these two functions?

Reply:
Done in a separate commit 👍